index.html

```diff
@@ -72,31 +72,31 @@ <h2>Challenger 2</h2>
             <h2>Latest Guess</h2>
             <h3 id="one-challenger">Challenger 1 name</h3>
```

Comment:
Words in html that are capitalized like in the Challenger 1 name and other text in the document should all be lowered case.  The way to format these words are though CSS where CSS can capitalize letters in way we want.  We do this so that all text in the html are consistent with each other by the formatting.

Reply:
Interesting, I had not heard that.  I will look out for that in the future.